### ISSUE TITLE
Segmentation Fault When Loading Remote Data

### ISSUE THREAD

**Issue by matthewniznik** [Unidata/netcdf4-python#492]
I use anaconda python on a MacBook Pro running OS X El Capitan. I updated my python packages via anaconda this morning and then re-added netCDF4 (1.1.9-np110py27_0).

Afterwards, when I call netCDF4.Dataset to load remote data I get a segmentation fault. Here's the simplest code that will lead to the segmentation fault:

netCDF4.Dataset('https://weather.rsmas.miami.edu/repository/opendap/synth:eab82de2-d682-4dc0-ba8b-2fac7746d269:L2FsbFZhcnNfcjkweDQ1XzMubmM0/entry.das','r')

I did not get the same error when using that call on a file on the local hard drive. If it helps, I've also attached the Mac error report generated when this happens in an IPython Notebook.

[errorReport.txt](https://github.com/Unidata/netcdf4-python/files/42926/errorReport.txt)

**Comment by jswhit** [comment]
Works on my mac.  Looks like a segfault in the C lib - perhaps a bug that has been fixed?  What version of the netcdf c lib are you using?  You can find out using:

    >>> import netCDF4
    >>> netCDF4.__version__
    '1.2.2'
    >>> netCDF4.__hdf5libversion__
    u'1.8.15-patch1'
    >>> netCDF4.__netcdf4libversion__
    u'4.4.0-development'

**Comment by matthewniznik** [comment]
Yeah, it does look like a few of them are out of date. Here's the output, and in the meantime I'll try updating the other libraries and see if that fixes things.

<code>In [6]: netCDF4.__version__</code>
<code>Out[6]: '1.1.9'</code>

<code>In [7]: netCDF4.__hdf5libversion__</code>
<code>Out[7]: u'1.8.15-patch1'</code>

<code>In [8]: netCDF4.__netcdf4libversion__</code>
<code>Out[8]: u'4.3.3.1'</code>

**Comment by jswhit** [comment]
I just tried here with netcdf 4.3.3.1 and hdf5 1.8.15-patch1, and it works.

**Comment by matthewniznik** [comment]
Just to note, downgrading to 1.1.8 worked for me as a temporary fix. I will try 1.2.2 soon.
